blocks/rapid-result/rapid-result.css

```diff
@@ -30,6 +30,11 @@
     min-height: 12.875rem;
 }
 
+.block.rapid-result .main-wrapper > ul.no-results {
+    display: block;
+    min-height: unset;
```

Comment:
@JiangLong2019  is this adding any layout shift ?

Reply:
@anagarwa I wasn't aware of anything about that, did you find anything? Actually, all blocks are following this way which was introduced by https://github.com/aemsites/icicidirect/pull/230. I missed this from yesterday's PR.

Reply:
I think there is no way we can avoid layout shifting as we are intentionally removing the empty space in the block in case of no results